src/lib/util/Config.ts

```diff
@@ -77,6 +104,8 @@ export class Config {
 		return this.configContent.javaHome;
 	}
 
+	// FIXME: Not supported yet - the logic is not hooked up to the actual call
+	// Leaving this as-is instead of moving to getConfigValue() style
 	public isEngineEnabled(name: string): boolean {
```

Comment:
Ideally, I'd want to fix this method to follow the new style as well. However, this would mean the default config will auto-populate disable = false in Config.json and will mislead the users to think that we support enable/disable. Disable code is only partially hooked up - this value wouldn't really disable the engine.
What remains is probably a simple check in DefaultRuleManager to ignore an engine if it is disabled. If you both feel this is a priority, we can create a separate work item to fix it the right way.

Reply:
I think waiting for 3.0 is a good idea.